kerastuner/tuners/bayesian.py

```diff
@@ -76,7 +76,10 @@ def __init__(self,
         self._tried_so_far = set()
         self._max_collisions = 20
         self._random_state = np.random.RandomState(self.seed)
-        self.gpr = gaussian_process.GaussianProcessRegressor(
+        self.gpr = self.make_gpr()
+        
+    def make_gpr(self):
```

Comment:
Let's make this private: `def _make_gpr`

Reply:
Should be good now.